capstone-frontend/src/components/SearchBookModal.jsx

```diff
@@ -128,6 +109,19 @@ export class SearchBookModal extends Component {
     }
   }
 
+  clubSubmit = async (gbookID) => {
+    let updateJson = {
+      id: this.props.objectId,
+      gbookID: gbookID,
+    }
+    await fetch(this.props.putURL, {
+      method: "PUT",
+      body: JSON.stringify(updateJson)
+    });
```

Comment:
Should we have a catch on this to handle any errors if the PUT doesn't work?

Reply:
Good call.